custom/icds/location_rationalization/views.py

```diff
@@ -0,0 +1,78 @@
+from django.contrib import messages
+from django.http import HttpResponse
+from django.utils.decorators import method_decorator
+from django.utils.functional import cached_property
+from django.utils.translation import ugettext as _
+from django.utils.translation import ugettext_noop
+
+from memoized import memoized
+
+from corehq import toggles
+from corehq.apps.domain.views.base import BaseDomainView
+from corehq.apps.locations.models import LocationType
+from corehq.util.files import safe_filename_header
+from corehq.util.workbook_json.excel import get_workbook
+from custom.icds.location_rationalization.dumper import Dumper
+from custom.icds.location_rationalization.forms import (
+    LocationRationalizationRequestForm,
+)
+from custom.icds.location_rationalization.parser import Parser
+
+
+@method_decorator([toggles.LOCATION_RATIONALIZATION.required_decorator()], name='dispatch')
+class BaseLocationRationalizationView(BaseDomainView):
+    section_name = ugettext_noop("Location Rationalization")
+
+    @property
+    def page_context(self):
+        context = {}
+        return context
+
+
+class ValidateRequestView(BaseLocationRationalizationView):
```

Comment:
How about just calling this `ValidationView` and generally dropping the use of "request"? We use "request" so often to refer to an http request that it's odd to see it here in the sense of "something being asked for."

Reply:
okay, update names for views and forms and the template names. 
Will update any place other than that as i find them.

Reply:
https://github.com/dimagi/commcare-hq/pull/26865/commits/33c80236959e64d07120378b8a9475c16a665579